data/crac-creation/crac-creator-cim/src/main/java/com/farao_community/farao/data/crac_creation/creator/cim/crac_creator/CimCracCreator.java

```diff
@@ -30,6 +31,7 @@ public class CimCracCreator implements CracCreator<CimCrac, CimCracCreationConte
     private Crac crac;
     private Network network;
     CimCracCreationContext creationContext;
+    private CimCracCreationParameters cimCracCreationParameters;
```

Comment:
is it really necessary to include in class members ? it could be a parameter in createRemedialActions

Reply:
Okay!